test/integration/StrykerJestRunnerSpec.ts

```diff
@@ -6,19 +6,31 @@ import { expect } from 'chai';
 import JestTestRunner from '../../src/JestTestRunner';
 import * as path from 'path';
 
+// Get the project root, we will be stub process.cwd later on
+const jestProjectRoot = process.cwd();
+
+// Needed for Jest in order to run tests
+process.env.BABEL_ENV = 'test';
+
 describe('Integration StrykerJestRunner', function () {
+  // Set timeout for integration tests to 10 seconds for travis
   this.timeout(10000);
-  const jestTestRunnerRoot = process.cwd();
-  const reactProjectRoot = path.join(jestTestRunnerRoot, 'testResources', 'reactProject');
-  const exampleProjectRoot = path.join(jestTestRunnerRoot, 'testResources', 'exampleProject');
 
   let jestConfigEditor: JestConfigEditor;
   let runOptions: RunnerOptions;
   let getProjectRootStub: sinon.SinonStub;
 
   let sandbox: sinon.SinonSandbox;
 
-  process.env.BABEL_ENV = 'test';
+  // Names of the tests in the example projects
+  const testNames = [
```

Comment:
It looks like you use these once, yet you describe them at a high level.

Reply:
I use `testNames` in two tests (the test with package.json and jest.config.js)